graph/config/cytoscape/cytoscape.go

```diff
@@ -438,3 +450,12 @@ func generateGroupCompoundNodes(appBox map[string][]*NodeData, nodes *[]*NodeWra
 		}
 	}
 }
+
+func calPrecisionDelta(rateVal float64) int {
+	delta := 0
+	if rateVal < 1 {
+		base := int(1 / rateVal)
```

Comment:
I know that at the moment this function is only called for a rate that is strictly positive, but nevertheless can you add a check `&& rateVal > 0` ? That would prevent bad surprises if it changes in the future

Reply:
check added